apps/librelingo_yaml_loader/tests/test_yaml_loader_convert_phrase.py

```diff
@@ -0,0 +1,82 @@
+"""
+this file contains tests of the funcion
+librelingo_yaml_loader.yaml_loader._convert_phrase
+"""
+import pytest
+
+from librelingo_types import Phrase
+from librelingo_yaml_loader.yaml_loader import _convert_phrase
+
+from librelingo_fakes import fakes
+
+
+def _alt_versions():
+    return "Alternative versions"
+
+
+def _alt_translations():
```

Comment:
what's the purpose of functions like these? to avoid string repetition?

I don't think it makes sense like this :thinking:

Reply:
Cf. [my comment](https://github.com/kantord/LibreLingo/pull/1955#discussion_r780655919). I think that is a good way to have a "read-only constant". Of course, if you will try hard enough, you can overwrite everything, but now you will at least have a warning.

Reply:
but I think it does not need to be a constant :thinking:

Reply:
This variable is only being read. So it should be a constant.

Reply:
It does not even need to be a variable. It's just a key of a dictionary in a JSON/YAML structure 🤔 

this should be more or less equivalent to a simple dot notation, but that's not really available for Python and would not work well anyway since some of the keys have spaces 🤔

I don't see this really as a constant or variable though. Or can you show me some example or argument why use this notation for these purposes?

I think if we are feeling really bad about using string literals for getting values of dicts then probably the alternative would be to create a class for each of these dictionary types and using objects of those class. But that also sounds like unnecessary complication just to get a value from a dict 🤔

Or else if you really don't want string literals, why not do this?

```python
def get_alternative_translations(phrase: YamlPhrase):
  return phrase["Alternative translations"]
```